server/src/main/java/com/objectcomputing/checkins/services/questions/QuestionController.java

```diff
@@ -0,0 +1,96 @@
+package com.objectcomputing.checkins.services.questions;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+@Controller("/questions")
+@Secured(SecurityRule.IS_ANONYMOUS)
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name="questions")
+public class QuestionController {
+
+    //    an endpoint is created for create
```

Comment:
Placeholder comments?

Reply:
yes, easier than flipping back and forth to the issue. They will be removed before final pr, I promise.